analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```diff
@@ -16,4 +16,13 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="1.3.2" />
   </ItemGroup>
+
+  <Target Name="CopyBinaries" AfterTargets="Build">
+    <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)\*.dll" />
+      <BinariesToCopy Include="$(OutputPath)\*.json" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BinariesToCopy)" DestinationFolder="$(BinariesFolder)\RuleDescriptorGenerator" />
```

Comment:
Nice. You may consider adding some of the tasks options to speed up the copy:

* SkipUnchangedFiles
* UseHardlinksIfPossible
* UseSymbolicLinksIfPossible

https://learn.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2022

We use symbolic links on peach to avoid copying the files to the artefact directory and it works quite nice.

Reply:
>2023-02-03T09:32:30.8762232Z [INFO]      [copy] Copying 355 files to C:\ProgramData\vsts-agent\1\s\sonar-vbnet-plugin\target\analyzer\org\sonar\plugins\vbnet
2023-02-03T09:32:31.7435422Z [INFO]      [copy] Copying 1 file to C:\ProgramData\vsts-agent\1\s\sonar-vbnet-plugin\target\analyzer\static


This exec happens between these two and is not logged. So copying 355 rspec files + these 20 DLLs takes less than a second -> not worth improving further.